src/components/track-changes/suggestion-list/SuggestionSnippet.tsx

```diff
@@ -7,61 +7,201 @@
+              nodeName: suggestion.node.type.spec.name,
+              content: getInlineFootnoteContent(doc, suggestion.attrs),
+            },
+            message: '',
+          }
+        } else if (suggestion.node.type === schema.nodes.footnote) {
+          console.log('suggestion.node', suggestion.node, suggestion)
+          return {
+            snippet: {
+              operation: changeOperationAlias(dataTracked.operation),
+              nodeName: suggestion.node.type.name,
+              content: getFootnoteText(view.state, suggestion.node),
+            },
+            message: '',
+          }
+        } else {
+          return {
+            snippet: null,
+            message: `${suggestion.node.type.name
+              .charAt(0)
+              .toUpperCase()}${suggestion.node.type.name.slice(1)} ${
+              dataTracked.operation
+            }`,
+          }
+        }
+      }
+
+      if (ChangeSet.isNodeAttrChange(suggestion)) {
+        if (suggestion.node.type === schema.nodes.inline_footnote) {
+          return {
+            snippet: {
+              operation: changeOperationAlias(dataTracked.operation),
+              nodeName: suggestion.node.type.spec.name,
+              content: getInlineFootnoteContent(doc, suggestion.newAttrs),
+            },
+            message: '',
+          }
+        } else if (suggestion.node.type === schema.nodes.footnote) {
+          return {
+            snippet: {
+              operation: changeOperationAlias(dataTracked.operation),
+              nodeName: suggestion.node.type.name,
+              content: getFootnoteText(view.state, suggestion.node),
+            },
+            message: '',
+          }
+        } else {
+          return {
+            snippet: null,
+            message: `${suggestion.node.type.name
+              .charAt(0)
+              .toUpperCase()}${suggestion.node.type.name.slice(1)} ${
+              dataTracked.operation
+            }`,
+          }
+        }
+      }
+
+      return { snippet: null, message: 'Unknown change!' }
     }
-    return 'Unknown change!'
-  }
+
+    const { snippet: newSnippet, message: newMessage } = getSnippetData()
+    setSnippet(newSnippet)
+    setMessage(newMessage)
+  }, [suggestion, doc, view.state, dataTracked.operation])
 
   return (
-    <>
-      <SnippetText isRejected={dataTracked.status === CHANGE_STATUS.rejected}>
-        {dataTracked.operation === CHANGE_OPERATION.delete ? (
-          <del>{changeTitle(suggestion)}</del>
-        ) : (
-          changeTitle(suggestion)
-        )}
-      </SnippetText>
-    </>
+    <SnippetText isRejected={dataTracked.status === CHANGE_STATUS.rejected}>
+      {snippet ? (
+        <>
+          <Operation color={dataTracked.operation}>
+            {snippet.operation}:
+          </Operation>
+          <NodeName>{snippet.nodeName}:</NodeName>
+          <Content>{parse(snippet.content)}</Content>
```

Comment:
Why do we need to parse that html into react tree? Are we going to work with it as with active react dom tree? Can we just insert html into ref?

Reply:
@mbartenev-atypon Why do we need to parse that html into react tree? Better for rendering and debugging.
Are we going to work with it as with active react dom tree? I don't think so.
Can we just insert html into ref? Yes, sure we can. But we have to ensure that the HTML content is sanitized, which I think we need it also for the current implantation.

I will keep this as note while implementing the rest nodes in the next sprint.

Reply:
Why do you think we need to santize it? Why it this better for debugging? Also in most cases it should be just a piece of text. Introducing a new dependency to parse a small piece of html is bit of an overkill. Additionally, whatever are the design requirement here, we should adjust them, if needed, to allow us to display the result here in a controlled way and not just dropping in html from the editor.

Reply:
Why do you think we need to sanities it?, because we are going to render whatever the user type, so if the user typed scripts it will be rendered.
Why it this better for debugging? it will be included in the react dom and tree.

Also in most cases it should be just a piece of text. Introducing a new dependency to parse a small piece of html is bit of an overkill. Additionally, whatever are the design requirement here, we should adjust them, if needed, to allow us to display the result here in a controlled way and not just dropping in html from the editor.
Totally agree, I was planing to do it by ref or css because its only needed for the alphabet order to match the style, so I will try to find a better way to render it in the next sprint without introducing any extra dependencies.